### ISSUE TITLE
npm list -g shows empty

### ISSUE THREAD

**Issue by eurekazheng** [coreybutler/nvm-windows#597]
**NVM4W version:** 1.1.7

### My Environment

- Windows 10 Home 1909, Chinese

### Expected Behavior

`npm list -g` should show what's inside the npm prefix node_modules (D:\Programs\nvm\v15.4.0\node_modules)

### Actual Behavior

`npm list -g` shows:
D:\Programs\nodejs -> D:\Programs\nvm\v15.4.0
`-- (empty)

### Steps to reproduce the problem:

```
nvm install latest
npm install -g vue
npm list -g
```
**These commands are in Powershell with administrative privileges.**

### More

**My npm config:** 
`npm config list`

```
; "user" config from C:\Users\eurekazheng\.npmrc

proxy = "http://127.0.0.1:7890/"
registry = "https://registry.npm.taobao.org/"

; "cli" config from command line options

omit = []
user-agent = "npm/7.0.15 node/v15.4.0 win32 x64"

; node bin location = D:\Programs\nodejs\node.exe
; cwd = D:\Programs\nvm\v15.4.0
; HOME = C:\Users\eurekazheng
; Run `npm config ls -l` to show all defaults.
```

I also tried making a soft link in default nodejs installation location:
`mklink /d C:\Users\eurekazheng\AppData\Roaming\nvm D:\Programs\nvm`
**Though this issue exists even when I install nvm-windows and node in default locations.**

Tried restarting the computer and reinstalling node.

I've viewed the common issues and am pretty sure that I've uninstalled the previous nodejs and purged residual files fully.

**Comment by eurekazheng** [comment]
It turns out to be an issue related to the latest version (15.4.0) of Node.js
It's fixed when I switched to the LTS version (14.15.3)
Not sure if it's also related to nvm-windows so I remain this issue open for your discretion.

**Comment by coreybutler** [comment]
Closing since this is an upstream issue. See https://github.com/coreybutler/nvm-windows/issues/579.
